stable/cert-manager/templates/certificate-crd.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: certificates.certmanager.k8s.io
+{{- if semverCompare ">=2.10-0" .Capabilities.TillerVersion.SemVer }}
```

Comment:
I noticed issues when adding the `helm.sh/hook: crd-install` hook with versions of Tiller that do not support the hook.